x/evm/keeper/utils.go

```diff
@@ -28,7 +27,7 @@ func (k Keeper) DeductTxCostsFromUserBalance(
 	// fetch sender account from signature
 	signerAcc, err := authante.GetSignerAcc(ctx, k.accountKeeper, msgEthTx.GetFrom())
 	if err != nil {
-		return nil, stacktrace.Propagate(err, "account not found for sender %s", msgEthTx.From)
+		return nil, err
```

Comment:
how about adding additional info
`account not found for sender..`

Reply:
yeah, kept original error message with sdkerrors.Wrapf.